pages/docs/reference/python/steps/parallel.mdx

```diff
@@ -34,3 +34,17 @@ async def fn(
         )
     )
 ```
+
+## Frequently asked questions
+
+### Do parallel steps work if I don't use `async` functions?
+
+Yes, parallel steps work with both `async` and non-`async` functions. Since our execution model uses a separate HTTP request for each step, threaded HTTP frameworks (e.g. Flask) will create a separate thread for each step.
```

Comment:
```suggestion
Yes, parallel steps work with both `async` and non-`async` functions. Since our execution model uses a separate HTTP request for each step, threaded HTTP frameworks (for example, Flask) will create a separate thread for each step.
```